### ISSUE TITLE
Telemetry won't work with a remote device

### ISSUE THREAD

**Issue by perezju** [catapult-project/catapult#3167]
I guess this is probably not a common use case, but it used to do the trick for me before while working from another office, but it no longer does.

Appears to be an issue with the way the new forwarding mechanism (TsProxy, etc.) works.

```
remote> adb kill-server
local> adb kill-server
local> adb start-server  # get <port> number
local> ssh -f -NR <port>:localhost:<port> remote.machine.name
remote> adb devices  # devices connected on local machine are now listed!
remote> tools/perf/run_benchmark --browser=android-browser <some_benchmark>
[...]
WARNING:root:Failed to create forwarder. Currently forwarded connections:
WARNING:root:  00b54cf412b1baef tcp:34838 localabstract:chrome_device_forwarder
WARNING:root:  
WARNING:root:Relevant device tcp sockets in use:
WARNING:root:Possibly relevant lsof entries:
[...]
```

As an aside, Telemetry also doesn't seem to place nicely with OSX anymore. 😞 

@nedn

**Comment by eakuefner** [comment]
Could be a Devil issue -- I think I recall that Devil only officially supports Linux as a host platform.

@jbudorick

**Comment by jbudorick** [comment]
Yeah, we don't officially support mac. If it works, great, but we don't make any claims.

That said, devil *should* work with the kind of port-forwarding you're trying to do.

**Comment by perezju** [comment]
Yes, pretty sure devil does work (well `adb` does work, and most of devil is a wrapper around `adb`). I think the problem arises because, somehow, the port forwarding thing I'm doing via ssh clashes with the port forwarding that Telemetry is trying to set up between the host and device.

**Comment by jbudorick** [comment]
Oh, yeah, that doesn't work. Telemetry and/or devil would need to be able to tunnel additional ports over SSH to support that.

(gtests that require the spawned_test_server face the same problem w/ running remotely.)
